cb-talent-development/employer-style-base
# EM-1242: Why CB Success Section (ESB)

**Purpose**
> As a potential buyer or high-level decision maker, I want to visit Hiring Dot and know what CareerBuilder can offer me as a vendor/partner.

This creates a mixin based off @arelia 's animated play button.

https://codepen.io/arelia/pen/wdWwym

**JIRA**
https://cb-content-enablement.atlassian.net/browse/EM-1242

**Changes**
* Improvements and fixes
  * N/A

* Changes to developer setup/environment
  * N/A

* Architectural changes
  * Adds `play-rippling-button` to `_button.scss`

* Migrations or Steps to Take on Production
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before

* After
![screen shot 2017-08-22 at 10 23 17 am](https://user-images.githubusercontent.com/5348098/29573323-4df576f2-8724-11e7-8eff-077660998a9d.png)

![screen shot 2017-08-22 at 10 23 36 am](https://user-images.githubusercontent.com/5348098/29573333-52f98a62-8724-11e7-9c95-fa3ba06fe568.png)

**Feature Server**
http://web.employer-6.development.c66.me/company/why-careerbuilder

**How to Verify These Changes**
* Specific pages to visit
  * In the Feature Server link, thumb / scroll to the Success section

* Steps to take
  * See if everything looks good, and compare it to the Desktop and Mobile Invision mocks in the Jira
  * Compare the play button to the Codepen link above
  * Hover over the play button, its color should change (based on Arelia's codepen)
  * Click on the play button, the Wistia video should play

* Responsive considerations
  * The close button should remain the same (size and width) all the way down to mobile

**Relevant PRs/Dependencies**
  * https://github.com/cbdr/employer/pull/966

**Additional Information**
N/A
